.github/workflows/publish.yaml

```diff
@@ -80,6 +84,7 @@ jobs:
       - lint
       - lint-client
       - test-client
+      - lint-server
```

Comment:
`lint-server` will need to run first, as it now has the potential to change `Types.ts` which could potentially break client side tests